sections/sigs.tex

```diff
@@ -2,83 +2,48 @@ \section{Special Interest Groups (SIGs)}
+  \item HackSIG\@: Hackathons
+  \item SIGINT\@: Cyber Security Group
+  \item LawTech: The Law and Technology Group
+  \item SIGnet: Computer Networking
+  \end{itemize}
 
-  \item The following points will be considered by the committee when processing
-    the application of a new SIG:-
-    \begin{enumerate}
-      \item The SIG should have a clearly defined goal, purpose or other reason
-      for existence deemed appropriate by the committee.
-      \item The SIG should have at least 4 members. In extraordinary circumstances
-      this can be overruled by the vote of 2/3 of the committee.
-      \item The SIG should have current CompSoc-member as a leader. The
-      leader shall be responsible for running the group and reporting back
-      to the Vice President and the CompSoc committee. It is encouraged
-      that the selection of a leader should be done democratically within the
-      group.
-    \end{enumerate}
+\item The following points will be considered by the committee when processing the application of a new SIG\@:
+  \begin{enumerate}
+  \item The SIG should have a clearly defined goal, purpose or other reason for existence deemed appropriate by the committee.
+  \item The SIG should have at least 4 members. In extraordinary circumstances this can be overruled by the vote of 2/3 of the committee.
+  \item The SIG should have current CompSoc-member as a leader. The leader shall be responsible for running the group and reporting back to the Vice President and the CompSoc committee. It is encouraged that the selection of a leader should be done democratically within the group.
+  \end{enumerate}
 
-  \item New SIGs must be approved by two thirds of the elected committee.
+\item New SIGs must be approved by two thirds of the elected committee.
 
-  \item Once an SIG is accepted:-
-    \begin{enumerate}
-      \item The leader of any accepted SIG will automatically have a seat and a
-        vote on the CompSoc committee.
-      \item is entirely up to the SIG how it wants to handle signing up members,
-        arranging meetings or other matters pertaining to the general
-        administration of the SIG.
-      \item The SIG has the right to receive support from the committee within
-        reason, including but not limited to financial support, creation of
-        advertising materials and free promotion on the CompSoc website,
-        newsletter and Facebook-group. Requests for such support should
-        be given in writing to the committee.
-      \item The leader of a SIG should continuously keep the committee informed
-        of the activities of the SIG.
-    \end{enumerate}
+\item Once an SIG is accepted:
+  \begin{enumerate}
+  \item The leader of any accepted SIG will automatically have a seat and a vote on the CompSoc committee.
+  \item is entirely up to the SIG how it wants to handle signing up members, arranging meetings or other matters pertaining to the general administration of the SIG\@.
```

Comment:
I know this wasn't present in the original to begin with, but could you add a `It` at the start of this sentence?

Reply:
Sure, done!